frontends/web/src/routes/account/send/utxos.tsx

```diff
@@ -128,6 +129,16 @@ export const UTXOs = ({
                       </span>
                       <AmountWithUnit alwaysShowAmounts amount={utxo.amount} convertToFiat/>
                     </div>
+                    <div className={style.date}>
+                      <span className={style.label}>
+                        {t('send.coincontrol.receivedDate')}:
```

Comment:
Reuse `transaction.details.date` here? Avoid adding more things to translate that we already have.